docs/.env.example

```diff
@@ -0,0 +1,3 @@
+DOCSEARCH_DOCS_APP_ID=
+DOCSEARCH_DOCS_API_KEY=
+DOCSEARCH_DOCS_INDEX_NAME=
```

Comment:
For my understanding, how is this filled out in prod?

Reply:
Good question! It's manually set in the Amplify app